survlearners/R/utils.R

```diff
@@ -61,7 +61,7 @@ sanitize_x = function(x){
 	if (!is.matrix(x) | !is.numeric(x) | any(is.na(x))) {
 		stop("x must be a numeric matrix with no missing values")
 	}
-	colnames(x) = stringr::str_c("covariate_", 1:ncol(x))
+	colnames(x) = paste0("covariate_", 1:ncol(x))

```

Comment:
You should also delete stringr from the  DESCRIPTION file

Reply:
ok, creating a new brach for it this time :)

Reply:
sounds good! when this repo grows in complexity you can realize doing many small changes is beneficial: it makes uncovering bugs/unanticipated behavior much easier !